### PR TITLE
chore: also remove the PowerShell prompt (`> `) when copying shell-session content.

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

* Add `> ` to the ignored prefixes.

* `yarn format:fix`.

### CODE REVIEW

File: website/src/theme/CodeBlock/CopyButton/index.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ export default function CopyButtonWrapper(props) {
   const updatedProps = { ...props };
   if (
     updatedProps?.code?.length > 2 &&
-    (updatedProps.code.substring(0, 2) === '$ ' || updatedProps.code.substring(0, 2) === '# ')
+    (updatedProps.code.substring(0, 2) === '$ ' ||
+      updatedProps.code.substring(0, 2) === '# ' ||
+      updatedProps.code.substring(0, 2) === '# ')
```
Comment (jeffmaury) [inline website/src/theme/CodeBlock/CopyButton/index.js]: I don't understand, seems code is duplicated

Comment (themr0c) [inline website/src/theme/CodeBlock/CopyButton/index.js]:
```suggestion
      updatedProps.code.substring(0, 2) === '> ')
```

Comment (themr0c) [reply website/src/theme/CodeBlock/CopyButton/index.js]: Indeed, my mistake

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**